publish/publish.proj

```diff
@@ -209,6 +209,26 @@
           DependsOnTargets="GetPackagesToSign;SignFiles">
   </Target>
 
+  <Target Name="PushSignedCoreHostPackagesToAzure">
+    <Message Text="Re-uploading signed CoreHost packages to Azure" />
```

Comment:
Why don't we sign the packages before we upload them to Azure in the first place?

Reply:
We can only sign on Windows, unfortunately.

Reply:
👎

Reply:
lol my sentiment exactly :)